test/browser/run-test.sh

```diff
@@ -24,6 +24,9 @@ git clone --depth=1 https://github.com/cockpit-project/bots
 # support running from clean git tree
 if [ ! -d node_modules/chrome-remote-interface ]; then
     npm install chrome-remote-interface sizzle
+else
+    # make sure we use the correct node_modules
+    ./tools/node-modules checkout
```

Comment:
I don't like this if/else hack much, but let's fix the Firefox breakage ASAP -- we can clean this up in a follow-up. Thank you!

Reply:
Ok, I think it's worse if it was always pulling node modules right? How do you propose we make it better, just always pull node modules?

Reply:
I meant that it shuold be like this:
```
[ -d node_modules/chrome-remote-interface ] ||  ./tools/node-modules checkout
```